AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_EntityReferenceAdded.py

```diff
@@ -119,33 +148,109 @@ def AtomEditorComponents_EntityReference_AddedToEntity():
         general.idle_wait_frames(1)
         Report.result(Tests.creation_redo, entity_reference_entity.exists())
 
-        # 5. Enter/Exit game mode.
+        # Entities for EntityIdReferences tests
+        test_1 = EditorEntity.create_editor_entity('test_1')
+        test_2 = EditorEntity.create_editor_entity('test_2')
+        test_3 = EditorEntity.create_editor_entity('test_3')
+
+        # 5. 'EntityIdReferences' is a container property
+        Report.result(
+            Tests.entity_id_references_is_container,
+            entity_reference_component.is_property_container(
+                AtomComponentProperties.entity_reference('EntityIdReferences')))
+
+        # 6. Append item to 'EntityIdReferences'
+        entity_reference_component.append_container_item(AtomComponentProperties.entity_reference('EntityIdReferences'),
+                                                         test_1.id)
+        Report.result(
+            Tests.container_append,
+            entity_reference_component.get_container_item(
+                AtomComponentProperties.entity_reference('EntityIdReferences'), 0) == test_1.id)
+
+        # 7. Add item to 'EntityIdReferences'
+        entity_reference_component.add_container_item(AtomComponentProperties.entity_reference('EntityIdReferences'),
```

Comment:
styling nitpick: i think this formatting is more clear:
```
entity_reference_component.add_container_item(
    AtomComponentProperties.entity_reference('EntityIdReferences'), 1, test_1.id)
```
we should be able to fit all the parameters on one line this way

Reply:
fixing style